model/src/main/proto/feedback_reporting.proto

```diff
@@ -325,7 +326,7 @@ message FeedbackReportingAppContext {
   // The text size the app is set to.
   ReadingTextSize text_size = 5;
   // The text language the app is set to.
-  AppLanguage text_language = 6;
+  OppiaLanguage text_language = 6;
```

Comment:
Hi Ben. changing OppiaLanguage index to 1 throws error below while rebuilding project.
Execution failed for task ':model:generateProto'.
> protoc: stdout: . stderr: feedback_reporting.proto:332:32: Field number 1 has already been used in "model.FeedbackReportingAppContext" by field "navigation_drawer".

Reply:
Hi @KevinGitonga, try a new sequentially greater index like 10, and put it at the bottom. Note Ben's comment about marking "6" as "reserved" -- look up usage of "reserved" in other proto files and follow the same pattern here. That guards the deprecated field from being used in the future.

Reply:
Thanks Sean.

Reply:
Using a sequentially greater index that is not use within the scope of a given proto message fixes the issue.

Reply:
Sounds good, thanks. Just be sure not to overwrite the current one -- mark it reserved instead and put the new one at the bottom.

Reply:
I have done it as below, hope this is right reserved 12 and used 16 which is greater and not used in the Profile message

message Profile {
  // Unique ID given to each profile.
  ProfileId id = 1;

  // Name of the user.
  string name = 2;

  // Determines whether profile is locked by a PIN.
  bool has_pin = 3;

  // PIN of the profile.
  string pin = 4;

  ProfileAvatar avatar = 5;

  // Determines whether profile is admin.
  bool is_admin = 6;

  // Determines whether profile can download content.
  bool allow_download_access = 7;

  // Represents the time the user was last logged in.
  int64 last_logged_in_timestamp_ms = 8;

  // Represents the time the user's profile was created.
  int64 date_created_timestamp_ms = 9;

  // Represents user selected reading-text-size.
  ReadingTextSize reading_text_size = 10;

  // Represents user selected audio-language.
  AudioLanguage audio_language = 11;

  // Reserve 12 which was used before as using it might cause import issues for older profiles.
  reserved 12;

  // Represents a generally study-unique ID that helps identify this profile for analytics logs that
  // are enabled as part of a user study. In non-user study cases this is expected to either be
  // empty or a meaningless value.
  string learner_id = 13;

  // Represents the number of times a user has successfully logged into their profile.
  int32 number_of_logins = 14;

  // Determines if the in-exploration continue button animation has been seen by the user or not.
  bool is_continue_button_animation_seen = 15;

  // Represents user selected content-language.
  OppiaLanguage oppia_language = 16;
}